agent/04_agent_configure.sh

```diff
@@ -27,19 +104,21 @@ function get_static_ips_and_macs() {
         static_ips=$NUM_MASTERS+$NUM_WORKERS
     fi
 
-    if [[ "$IP_STACK" = "v4" ]]; then
-        external_subnet=$EXTERNAL_SUBNET_V4
-    else
-        external_subnet=$EXTERNAL_SUBNET_V6
-    fi
-
     # Set outside the range used for dhcp
     base_ip=80
 
     for (( i=0; i<${static_ips}; i++ ))
     do
         ip=${base_ip}+${i}
-        AGENT_NODES_IPS+=($(nth_ip ${external_subnet} ${ip}))
+        if [[ "$IP_STACK" = "v4" ]]; then
+            AGENT_NODES_IPS+=($(nth_ip ${EXTERNAL_SUBNET_V4} ${ip}))
+        elif [[ "$IP_STACK" = "v6" ]]; then
+            AGENT_NODES_IPS+=($(nth_ip ${EXTERNAL_SUBNET_V6} ${ip}))
```

Comment:
For consistency here I'd expect `AGENT_NODES_IPSV6`

Reply:
Changed to make this more consistent.